.github/workflows/ci.yaml

```diff
@@ -130,3 +130,34 @@ jobs:
         with:
           name: manager_linux
           path: convenient_test_manager.tar.gz
+
+  build_manager_headless_cli:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: subosito/flutter-action@v2
+        with:
+          channel: stable
+          architecture: x64
+
+      - name: Install dependencies
+        run: dart pub get
+        working-directory: packages/convenient_test_manager_dart
+
+      - name: Dart Compile
+        run: |
+          dart compile exe ./bin/convenient_test_manager_dart.dart -o manager_headless
```

Comment:
Looks like we can `dart compile exe` on ubuntu, mac, and windows (using the `runs-on` param). Then we can native executable for all three platforms :)